src/System.Runtime.Analyzers/Core/InitializeStaticFieldsInline.Fixer.cs

```diff
@@ -27,10 +29,10 @@ public sealed override FixAllProvider GetFixAllProvider()
         }
 
         public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
-        {                              
+        {
+            // TODO: Implement the fixer.
             // This is to get rid of warning CS1998, please remove when implementing this analyzer
             await new Task(() => { });
```

Comment:
I am planning to delete the fixer file completely, not sure if it is worth writing a fixer for this.